### DISCUSSION TITLE
Standup (Wednesday, October 26th)

### DISCUSSION THREAD

**Discussion by antonieto** [BBGTC/food-bank-app#22] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Added react-native-elements to project
 - Started working on styling for textfields, headers and buttons

**What are your priorities for the next 24 hours?**:
 - Keep working on styles

**Blockers**:
 - rneui isn't working correctly

**Shoutouts**:
 - @BrandonMagana for being really hardworking

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Read about **django channels**, will be helpful when we build the QR feature.

**What are your priorities for the next 24 hours?**:
 - Address @BrandonMagana comments on #16 

**Blockers**:
 - Semana TEC, although it's finished now

**Shoutouts**:
 - @BrandonMagana for interviewing today
